apps/zotonic_core/src/support/z_mqtt_sessions_runtime.erl

```diff
@@ -97,13 +107,26 @@ maybe_set_language(#{ <<"preferences">> := #{ <<"language">> := Lang } }, Contex
         error:undef ->
             z_context:set_language(Lang, Context)
     end;
+maybe_set_language(#{ language := <<>> }, Context) ->
+    Context;
+maybe_set_language(#{ language := Lang }, Context) ->
+    try
```

Comment:
@mworrell Is this construct still needed? It looks like the function is always defined.

Reply:
I suspect it was a protection against illegal timezones.

Reply:
Are the language/timezone not always in the preferences?

Reply:
mod_translation could also be switched off. But in that case mod_translation is still callable. No `{error, undef}` is returned in that case.